testing/src/test/java/org/motechproject/nms/testing/it/IntegrationTests.java

```diff
@@ -77,10 +77,11 @@
     /**
      * Kilkari
      */
+    RchBeneficiaryImportServiceBundleIT.class,
```

Comment:
This commit looks totally unnecessary - always commt from IDE after checking the differences manually